db/common_test.go

```diff
@@ -0,0 +1,143 @@
+// Package db - common test
+// The following file contains all the common functions for testing.
+package db
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"testing"
+
+	"github.com/packethost/pkg/log"
+)
+
+var (
+	ids = testID{
+		instanceID: "da013837-a30c-237a-adfc-2d9ad69d5000",
+		hardwareID: "ce2e62ed-826f-4485-a39f-a82bb74338e2",
+		templateID: "da013837-a30c-237a-adfc-2d9ad69d5011",
+		workflowID: "b356eede-22c1-11eb-8c17-0242ac120004",
+	}
+)
+
+func newCTX() ctx {
+	c, closer := context.WithCancel(context.Background())
+	return ctx{
+		context: c,
+		cancel:  closer,
+	}
+}
+
+func (a *testArgs) setCTX() {
+	a.ctx = newCTX()
+}
```

Comment:
Not sure the value of this. setCTX is only called from one place so we can just use newCTX there. On top of that though ctx.context is whats always being used so might as well just make it the Context instead. We aren't really gaining much by abstracting a call to `ctx, _ := context.WithCancel`.

Also is `cancel()` never called right? I think thats probably fine since its just for short running tests anyway, but if so then there's even less of a need for a ctx struct.